prover/setup.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+cd ..
+export ZKSYNC_HOME=$(pwd)
```

Comment:
setting this environment variable here is 'surprising' - let's check its value instead and complain if this is set incorrectly.

Reply:
Fixed here [e7b613f](https://github.com/matter-labs/zksync-era/pull/258/commits/e7b613f39fd439668bada0b8ee988970b39bef54)